File: src/main/java/com/vspk/bookrest/BookrestApplication.java
package com.vspk.bookrest;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableAsync;

@EnableAsync
@SpringBootApplication
public class BookrestApplication {

    public static void main(String[] args) {
        SpringApplication.run(BookrestApplication.class, args);
    }

}

File: src/main/java/com/vspk/bookrest/BookrestApplication.kt
package com.vspk.bookrest;

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableAsync

@EnableAsync
@SpringBootApplication
open class BookrestApplication

fun main(args: Array<String>) {
    runApplication<BookrestApplication>(*args)
}
